src/main/flight/pid.c

```diff
@@ -886,8 +886,12 @@ static void detectAndSetCrashRecovery(
                 && fabsf(delta) > crashDtermThreshold
                 && fabsf(errorRate) > crashGyroThreshold
                 && fabsf(getSetpointRate(axis)) < crashSetpointThreshold) {
-                inCrashRecoveryMode = true;
-                crashDetectedAtUs = currentTimeUs;
+                if (crash_recovery == PID_CRASH_RECOVERY_DISARM) {
+                    disarm();
```

Comment:
We need `setArmingDisabled(ARMING_DISABLED_CRASH_DETECTED)` here.

Reply:
Gah, that file was still open and unsaved in vim.